M/MUSCLE/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+using BinaryBuilder, Pkg
+
+name = "MUSCLE"
+version = v"5.2"
+
+sources = [
+    GitSource("https://github.com/rcedgar/muscle.git",
+                  "6c601163998616bb88991931e443c645858e162c"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/muscle/src/
+atomic_patch ${WORKSPACE}/srcdir/patches/remove-ffast-math.patch
+make -j${nproc}
+install -D -m 755 ${WORKSPACE}/srcdir/muscle/src/Linux/./muscle ${bindir}/./muscle
```

Comment:
Make it verbose so we have a record of what is being moved where
```suggestion
install -Dvm 755 "Linux/muscle" "${bindir}/muscle"
```